fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -279,6 +280,9 @@ def apply_interface_data(self, interface_node, port):
                                     port.trunk_vlans = []
                                 port.trunk_vlans += parse_range(member.text)
 
+            if port.trunk_native_vlan is None :
```

Comment:
I don't get this IF

it looks like you are updating a port... if the port already had a native-vlan and the configuration changes it then it will be ignored.

I suggest adding this test:

set native vlan 1000
set native vlan 2000
assert native vlan is 2000